packages/theme/components/Category/CategorySidebar.vue

```diff
@@ -1,83 +1,64 @@
 <template>
-  <div>
-    <slot
-      v-if="!$fetchState.pending"
-      name="content"
-      :category-tree="categoryTree"
-      :active-category="activeCategory"
+  <SfAccordion
+    :open="activeCategory"
+    :show-chevron="true"
+  >
+    <SfAccordionItem
+      v-for="(cat, i) in categoryTree.items"
+      :key="i"
+      :header="cat.label"
     >
-      <SfAccordion
-        :open="activeCategory"
-        :show-chevron="true"
-      >
-        <SfAccordionItem
-          v-for="(cat, i) in categoryTree.items"
-          :key="i"
-          :header="cat.label"
+      <SfList class="list">
+        <SfListItem
+          v-for="(subCat, j) in cat.items"
+          :key="j"
+          class="list__item"
         >
-          <SfList class="list">
-            <SfListItem
-              v-for="(subCat, j) in cat.items"
-              :key="j"
-              class="list__item"
-            >
-              <SfMenuItem
-                :count="subCat.count || ''"
-                :label="subCat.label"
+          <SfMenuItem
+            :count="subCat.count || ''"
+            :label="subCat.label"
+          >
+            <template #label="{ label }">
+              <nuxt-link
+                :to="localePath(getAgnosticCatLink(subCat))"
+                :class="subCat.isCurrent ? 'sidebar--cat-selected' : ''"
               >
-                <template #label="{ label }">
-                  <nuxt-link
-                    :to="localePath(getAgnosticCatLink(subCat))"
-                    :class="subCat.isCurrent ? 'sidebar--cat-selected' : ''"
-                  >
-                    {{ label }}
-                  </nuxt-link>
-                </template>
-              </SfMenuItem>
-              <SfMenuItem
-                v-for="(subSubCat, z) in subCat.items && subCat.items"
-                :key="z"
-                :count="subSubCat.count || ''"
-                :label="subSubCat.label"
-                class="list__item__sub"
+                {{ label }}
+              </nuxt-link>
+            </template>
+          </SfMenuItem>
+          <SfMenuItem
+            v-for="(subSubCat, z) in subCat.items && subCat.items"
+            :key="z"
+            :count="subSubCat.count || ''"
+            :label="subSubCat.label"
+            class="list__item__sub"
+          >
+            <template #label="{ label }">
+              <nuxt-link
+                :to="localePath(getAgnosticCatLink(subSubCat))"
+                :class="subSubCat.isCurrent ? 'sidebar--cat-selected' : ''"
```

Comment:
```suggestion
                :class="{'sidebar--cat-selected': subSubCat.isCurrent}"
```

super nitpick :P ignore if you want

Reply:
fixed